### PR TITLE
removed dead urls 

### PR DESCRIPTION
**Pull request by Trilarion**
They are either dead since quite some time or redirect to some unrelated site. I removed them and also some screenshots (they are not cached, are they?). Please check in case I made a mistake.

See also related discussion in https://github.com/opengaming/osgameclones/issues/856

### CODE REVIEW

File: games/g.yaml [hunk 1]
```diff
@@ -418,5 +417,4 @@
   - Hexen
   repo: https://github.com/coelckers/gzdoom
   type: remake
-  updated: 2015-04-03
-  url: http://www.osnanet.de/c.oelckers/gzdoom/
```
Comment (cxong) [inline games/g.yaml]: https://zdoom.org/index

File: games/i.yaml [hunk 2]
```diff
@@ -170,8 +169,7 @@
   - Iron Seed
   repo: http://www.ironseed.com/ironseed-v1.20.0016-2013-03-17.zip
```
Comment (cxong) [inline games/i.yaml]: this is also gone - let's remove the whole game

File: games/p.yaml [hunk 3]
```diff
@@ -74,8 +74,7 @@
   - Buster Bros
   repo: http://pangzero.cvs.sourceforge.net/pangzero/pangzero/
   type: remake
-  updated: 2015-04-18
-  url: http://apocalypse.rulez.org/pangzero
```
Comment (cxong) [inline games/p.yaml]: https://apocalypse.rulez.org/kozos/UpiPangBlog

File: games/r.yaml [hunk 4]
```diff
@@ -538,6 +533,5 @@
   - Ryzom
   repo: https://bitbucket.org/ryzom/ryzomcore/overview
   type: remake
-  updated: 2015-04-13
-  url: http://www.ryzomcore.org/
```
Comment (cxong) [inline games/r.yaml]: https://ryzom.com

File: games/g.yaml (hunk 1 above)
Comment (Trilarion) [reply games/g.yaml]: looks good

File: games/i.yaml (hunk 2 above)
Comment (Trilarion) [reply games/i.yaml]: you could still access it through the archive https://web.archive.org/web/20150725164943/http://www.ironseed.com/ironseed-v1.20.0016-2013-03-17.zip
delete or keep?

File: games/p.yaml (hunk 3 above)
Comment (Trilarion) [reply games/p.yaml]: I changed it, but this blog is in Hungarian, might be difficult to understand by most visitors

File: games/r.yaml (hunk 4 above)
Comment (Trilarion) [reply games/r.yaml]: changed

File: games/i.yaml (hunk 2 above)
Comment (cxong) [reply games/i.yaml]: ok; linking to the web archive is fine too

Comment (Trilarion) [reply games/i.yaml]: did it

**Comment by Trilarion** [comment]
From my side ok to merge.

**Pull request merged**